### ISSUE TITLE
Running as docker container  throws " An assembly specified in the application dependencies manifest (Asp2017.deps.json) was not found"

### ISSUE THREAD

**Issue by JinnaBalu** [MarkPieszak/aspnetcore-angular2-universal#486]
I have followed this steps to create Docker image and run as 

### Publish

```bash
dotnet publish -c Release
```

### Created ***Dockerfile*** in bin/Release/netcoreapp2.0/publish with 

```yml
FROM microsoft/dotnet:2.0-runtime
WORKDIR /app
COPY . .
CMD ASPNETCORE_URLS=http://*:$PORT dotnet Asp2017.dll

```

### Created Docker image with 

```bash
docker build -t dotnetcoreapi ./bin/Release/netcoreapp2.0/publish
```

### Running using

```bash
docker run -p 8009:5000 dotnetcoreapi
```

## ERROR
```txt
An assembly specified in the application dependencies manifest (Asp2017.deps.json) was not found:
    package: 'Microsoft.ApplicationInsights.AspNetCore', version: '2.1.1'
    path: 'lib/netstandard1.6/Microsoft.ApplicationInsights.AspNetCore.dll'
  This assembly was expected to be in the local runtime store as the application was published using the following target manifest files:
    aspnetcore-store-2.0.0-linux-x64.xml;aspnetcore-store-2.0.0-osx-x64.xml;aspnetcore-store-2.0.0-win7-x64.xml;aspnetcore-store-2.0.0-win7-x86.xml
```
![image](https://user-images.githubusercontent.com/11784253/32994549-f60cc06a-cd8e-11e7-81d7-6bc2bbcdad04.png)

**Comment by Crazyht** [comment]
Hi, 

It's work weel with this docker file : 
```
FROM microsoft/aspnetcore:2.0

RUN apt-get update
RUN apt-get install apt-utils wget -y    
RUN apt-get install -y --no-install-recommends gnupg2
RUN curl -sL https://deb.nodesource.com/setup_6.x | bash -
RUN apt-get install -y nodejs

ENV ASPNETCORE_URLS "http://0.0.0.0:80"
ENV ASPNETCORE_ENVIRONMENT Production

WORKDIR /app

COPY . .

RUN npm install

EXPOSE 80:80
ENTRYPOINT ["dotnet", "Asp2017.dll"]
```

**Comment by JinnaBalu** [comment]
@Crazyht  thank you for the reply. Where do i need to place this docker file in root or in publish flder?

**Comment by Crazyht** [comment]
In my case dockerfile is copied to publish folder, else you need adapt COPY . . line to according source folder

**Comment by JinnaBalu** [comment]
I have tried using the configuration which you gave here. Still i am facing some other issue.

![image](https://user-images.githubusercontent.com/11784253/33043341-2050cae2-ce6b-11e7-9a27-7e0eaac36904.png)
Able to create image and run, but when i try to access it from browser will get this error.

**Comment by Crazyht** [comment]
it's the same error : https://github.com/MarkPieszak/aspnetcore-angular2-universal/issues/482 ? No ?

**Comment by MarkPieszak** [comment]
We should be all set now that 5.0 production builds are fixed (and merged into Master) via #538
